<ui version="4.0" >
 <class>AddDistroDialog</class>
 <widget class="QDialog" name="AddDistroDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addDistroOkButton" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>360</y>
     <width>94</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text" >
    <string>Setup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addDistroCancelButton" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>360</y>
     <width>94</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listNonSetupDistrosWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>260</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="1" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Associated OSCAR repository</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLineEdit" name="oscarRepoEdit" />
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Distribution repository</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLineEdit" name="distroRepoEdit" />
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addDistroCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddDistroDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>456</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel" >
     <x>263</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
